.travis.yml

```diff
@@ -16,19 +21,10 @@ node_js:
   - "0.6"
   - "0.4"
 before_install:
-  - '[ "${TRAVIS_NODE_VERSION}" == "0.6" ] || npm install -g npm@~1.4.6'
+  - '[ "${TRAVIS_NODE_VERSION}" == "0.6" ] || [ "${TRAVIS_NODE_VERSION}" == "v5.0" ] || npm install -g npm@~1.4.6'
```

Comment:
I've found [this combination](https://github.com/ljharb/is-callable/blob/master/.travis.yml#L34-L36) to be most effective.